nucliadb/src/nucliadb/ingest/fields/base.py

```diff
@@ -208,27 +208,50 @@ async def set_error(self, error: Error) -> None:
             error=error,
         )
 
-    async def get_question_answers(self) -> Optional[QuestionAnswers]:
-        if self.question_answers is None:
+    async def get_question_answers(self, force=False) -> Optional[FieldQuestionAnswers]:
+        if self.question_answers is None or force:
             sf = self.get_storage_field(FieldTypes.QUESTION_ANSWERS)
-            payload = await self.storage.download_pb(sf, QuestionAnswers)
+            payload = await self.storage.download_pb(sf, FieldQuestionAnswers)
```

Comment:
Could you add a warning here if the operation fails. We can use it to find and delete any KB with the old Q&A implementation